pkg/ingester/tailer.go

```diff
@@ -147,7 +147,7 @@ func (t *tailer) processStream(stream logproto.Stream, lbs labels.Labels) []*log
 
 	sp := t.pipeline.ForStream(lbs)
```

Comment:
what about tailing ?

Reply:
This is just a question btw, not asking to do anything for this PR.

Reply:
Missed it altogether. Good call.

We have feature flags setup, now, so this can be a follow in PR right after this one.